docs_versioned_docs/version-ros2jazzy/ros/config/yaml/platform/controller.mdx

```diff
@@ -17,3 +17,53 @@ Switching between each is done by setting the `controller` parameter under the `
 ```yaml
 controller: ps4 # one of ps4, ps4, logitech, xbox
 ```
+
+## Teleop Speeds
+
+By default, holding the left trigger button of the controller enables driving at "normal" speeds
+while holding the right trigger button enables driving at "turbo" speeds.
+
+The table below shows the default linear and angular speeds for each platform:
+
+| Platform   | Linear speed (normal, m/s) | Linear speed (turbo, m/s) | Angular speed (normal, rad/s) | Angular speed (turbo, rad/s) |
+|:---------- |:-------------------------- |:------------------------- |:----------------------------- |:---------------------------- |
+| Husky A300 | 0.4                        | 1.5                       | 0.5                           | 1.0                          |
+| Husky A200 | 0.4                        | 1.0                       | 0.6                           | 1.2                          |
+| Jackal     | 0.4                        | 2.0                       | 0.6                           | 1.4                          |
+| Warthog    | 0.5                        | 1.0                       | 2.4                           | 2.4                          |
+| Dingo-D    | 0.4                        | 1.0                       | 0.6                           | 1.0                          |
+| Dingo-O    | 0.4                        | 1.0                       | 0.5                           | 1.4                          |
+| Ridgeback  | 0.4                        | 0.4                       | 0.5                           | 0.5                          |
+
+### Changing Teleop Speeds
+
+The normal and turbo teleop speeds can be adjusted by modifying the controller node's ROS parameters by using the
+[`extras.ros_parameters`](extras) field in `robot.yaml`.
```

Comment:
Is this a broken hyperlink?
[`extras.ros_parameters`](extras)

Reply:
No, it's not broken. It links to the Platform > Extras page. The link works when I build the page locally.